sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -948,10 +964,7 @@ export class MessageReceiver extends LinkEntity {
             }),
           connectionId: connectionId,
           operationType: RetryOperationType.receiverLink,
-          retryOptions: {
-            maxRetries: Constants.defaultMaxRetriesForConnection,
-            retryDelayInMs: 15000
-          },
+          retryOptions: this._retryOptions,
```

Comment:
What is the default if a customer doesn't provide one? Is it still 150 times?

Reply:
Default is 3.
(All defaults are set in core-amqp)